src/containers/CartPage.less

```diff
@@ -0,0 +1,275 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.cart-container{
+  min-height: 100%;
+  margin-bottom: 0px;
+  :after {
```

Comment:
You are missing an `&`. You need to write `&:after` in order to define `:after` pseudoclass for `.cart-container`. That being said, if it looks as expected without it you might as well remove it as it is not being applied right now.